docs/glossary.rst

```diff
@@ -83,11 +83,26 @@ You can find links to our detailed documentations for
 
 Subgroup analysis
 ------------------------------------
-Subgroup analysis in ExaAn will select subgroup (which is a segment of data) based on the input argument, and then perform a regular delta analysis per subgroup as described before. 
+Subgroup analysis in ExaAn will select subgroup (which is a segment of data) based on the input argument, and then perform a regular delta analysis per subgroup as described before.
```

Comment:
sorry there's just a typo here (ExpAn)

Reply:
no, there is no typo, I don't know why it was actually committed, I tried to revert but it removes a space in the end all the time.

Reply:
ah, ok, will fix =)